Automattic/woocommerce-payments
# deps: add yarnhook

#### Changes proposed in this Pull Request









It can happen that we add/remove dependencies between merges.
But it can also happen that not everyone remembers to run `npm install` after new updates.

`yarnhook` detects changes on `package-lock.json` to determine whether or not we need to run `npm install` after checking out a new branch or updating our local branch.

It doesn't mean that it can fail - in my experience it might not run when rebasing or resolving merge conflicts. But it's a helpful tool, I think.

https://github.com/frontsideair/yarnhook

#### Testing instructions





* Check out this branch
* Run `npm install` to get `yarnhook`
* Run `git checkout -b test-deps && npm install --save-dev addition && git add . && git commit -am "test add"` to add a random dependency to a new temp branch
* Run `git checkout frosso/yarnhook`
* You should see the dependencies update running (because of the change of branch)
* Run `git branch -D test-deps` to delete the temp branch

-------------------

Don't think it's necessary/worth mentioning publicly, just a dev tool.
